opm/simulators/timestepping/AdaptiveSimulatorTimer.cpp

```diff
@@ -60,6 +60,7 @@ AdaptiveSimulatorTimer& AdaptiveSimulatorTimer::operator++ ()
     {
         ++current_step_;
         current_time_ += dt_;
+	assert(dt_ > 0);
```

Comment:
Lots of broken indentation.

Reply:
Yes, I see that now. Mostly due to use of tabs. I'll add a whitespace cleanup commit.